src/Entity/ShipmentTotalWeight.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+use Ups\Entity\UnitOfMeasurement;
+
+class ShipmentTotalWeight implements NodeInterface
+{
+    private $unitOfMeasurement;
+    private $Weight;
+
+    public function __construct($response = null)
+    {
+        if (null !== $response) {
+            if (isset($response->unitOfMeasurement)) {
+                $this->setUnitOfMeasurement($response->unitOfMeasurement);
+            }
+            if (isset($response->Weight)) {
+                $this->setWeight($response->Weight);
```

Comment:
Not sure this one is correct.

Reply:
It should be. It's taken from a prior entity (src/shipmentweight)